config/initializers/active_job.rb

```diff
@@ -1,5 +1,11 @@
 ActiveJob::Base.queue_adapter = if Rails.env.test?
                                   :test
+                                elsif Rails.env.development? || Figaro.env.pt_mode == 'on'
+                                  # letter_opener_web (used during development only)
+                                  # doesn't currently work with sidekiq running in a separate
+                                  # container since it relies on a shared ./tmp directory.
+                                  # https://github.com/fgrehm/letter_opener_web/issues/35
+                                  :inline
                                 else
```

Comment:
I'm fine with waiting for another PR, as long as it's fairly soon. I verified that emails are delivered on Mac via letter_opener. I haven't tried the Docker setup yet. Will do tomorrow.